.github/CODEOWNERS

```diff
@@ -70,20 +70,23 @@ ui/selectors/institutional           @MetaMask/mmi
 ui/components/component-library      @MetaMask/design-system-engineers
 
 # The Notifications team is responsible for code related to notifications,
-# authentication, and profile syncing inside the Extension.
 
 # Controllers
-**/controllers/authentication/**                    @MetaMask/notifications
 **/controllers/metamask-notifications/**            @MetaMask/notifications
 **/controllers/push-platform-notifications/**       @MetaMask/notifications
-**/controllers/user-storage/**                      @MetaMask/notifications
 
```

Comment:
These two controllers have been moved to the external package `profile-sync-controller`